### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by WashingtonKK**

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -29,8 +29,7 @@ jobs:
 
       - name: Build
         run: |
-          go build -mod=vendor -ldflags="-linkmode=external -extldflags=-static -s -w" cmd/agent/main.go
-          go build -mod=vendor -ldflags "-s -w" cmd/cli/main.go
+          go build -v ./...
```
Comment (drasko) [inline .github/workflows/main.yaml:32]: Is there a particular reason why you are changing linker flags?

**Review by drasko — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yaml (hunk 1 above)
Comment (WashingtonKK) [reply .github/workflows/main.yaml:32]: No, I replicated what was done on prism.

**Review by drasko — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yaml [hunk 2]
```diff
@@ -29,7 +29,15 @@ jobs:
 
       - name: Build
         run: |
-          go build -v ./...
+          make agent
```
Comment (drasko) [inline .github/workflows/main.yaml:32]: There is no particular reason to make service by service in CI - please use just `make` to make all services at once.

**Review by drasko — CHANGES_REQUESTED** [review]

File: .github/workflows/main.yaml (hunk 2 above)
Comment (WashingtonKK) [reply .github/workflows/main.yaml]: It is fine now

**Review by drasko — APPROVED** [review]
Approving, but we should consider to add `test` target to `Makefile` as well. That would be the most elegant.

**Pull request merged**